specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/communityGallery.json

```diff
@@ -297,15 +315,64 @@
       },
       "description": "The identifier information of community gallery."
     },
+    "CommunityGalleryMetadata": {
+      "type": "object",
+      "properties": {
+        "publisherUri": {
+          "type": "string",
+          "description": "The publisher uri of this community gallery."
+        },
+        "publisherContact": {
+          "type": "string",
+          "description": "The publisher contact of this community gallery."
+        },
+        "eula": {
+          "type": "string",
+          "description": "The eula of this community gallery."
+        },
+        "publicNames": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of public names the gallery has."
+        },
+        "privacyStatementUri": {
+          "type": "string",
+          "description": "The privacyStatementUri of this community gallery."
```

Comment:
Prefer to use plain english language words explaining the meaning of the thing, instead of just repeating the symbolic name